portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -89,7 +81,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       BlogHashMap blogMap = new BlogHashMap();
       blogMap.putInMap(messages);
 
-      // Load messages from BlogHashMap and respond with gson.
+      // If (user loads all BlogMessages) 
+      LinkedList<BlogMessage> allBlogMessages = blogMap.getMessages();
```

Comment:
This uses an old version of the BlogHashMap API, please use the current public method. If you think something is missing from the API comments, please ask Andrew to clarify them.

There are also a few below I noted.